src/RA_AchievementSet.cpp

```diff
@@ -310,59 +312,62 @@ BOOL AchievementSet::SaveToFile()
     //}
 }
 
-BOOL AchievementSet::Serialize(FileStream& Stream)
-{
-    //	Why not submit each ach straight to cloud?
-    return FALSE;
 
-    //FILE* pf = nullptr;
-    //const std::string sFilename = GetAchievementSetFilename( m_nGameID );
-    //if( fopen_s( &pf, sFilename.c_str(), "wb" ) == 0 )
+//BOOL AchievementSet::Serialize( FileStream& Stream )
```

Comment:
Should this function be gone or just marked as deprecated? Since I heard before commented code should not be committed. Should the currently commented out code be gone?